android/src/main/java/com/rnziparchive/RNZipArchivePackage.java

```diff
@@ -16,6 +16,12 @@
     modules.add(new RNZipArchiveModule(reactContext));
     return modules;
   }
+  
+  // deprecated >= RN 0.47.0
```

Comment:
I don't know any more details than mentioned in the commits here: https://github.com/facebook/react-native/releases/tag/v0.47.0 - it looks little bit schizophrenic, because the method was quite recently added and now they removed it. I removed the method completely first, but later realized it's better to remove just the `@Override` annotation and leave the method itself to make the code compatible with <0.47 RN versions too.